pkg/model/test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package model
+
+import "github.com/okteto/okteto/pkg/env"
+
+type Test struct {
+	Image   string   `json:"image,omitempty" yaml:"image,omitempty"`
+	Command []string `json:"command,omitempty" yaml:"command,omitempty"`
+}
+
+func (test *Test) expandEnvVars() error {
+	var err error
+	if len(test.Image) > 0 {
+		test.Image, err = env.ExpandEnv(test.Image)
+		if err != nil {
+			return err
+		}
+	}
+	if len(test.Command) > 0 {
+		for i, v := range test.Command {
+			test.Command[i], err = env.ExpandEnv(v)
```

Comment:
I think we don't have to expand variables of the commands, right? That should be expanded on runtime when commands are executed, right?

Reply:
Removed